### ISSUE TITLE
Can't display the value scope error

### ISSUE THREAD

**Issue by lwx5924** [overtrue/phplint#87]
Can't display the value scope error, code as below:

```php
<?php

function test() {

  $isok=false;

  if ($isok ) {
    $v = array(1,2,3);
    print("true\n");
    print( "v=". json_encode($v));

  } else {
    print("false\n");

    print( "v=". json_encode($v));  // here is no value v
  }
}

test();
```

but this report success:

```bash
phplint 2.0.2 by overtrue and contributors.

No config file loaded.

.

Time: < 1 sec   Memory: 2.0 MiB Cache: Yes

OK! (Files: 1, Success: 1)
```

**Comment by overtrue** [comment]
这属于运行时错误，不属于语法错误，所以无法检测，如果想要检查这种错误可以考虑 https://github.com/phpstan/phpstan

```bash
➜ php -l test.php
No syntax errors detected in a.php
```
